### PR TITLE
model payout restructuring

### PR DESCRIPTION
**Pull request by dhruv-divtech**

### CODE REVIEW

File: src/views/protectedDashboardViews/payoutsAndInvoicesTable/billingTable/BillingTable.tsx [hunk 1]
```diff
@@ -2,132 +2,30 @@ import TableContainer from '@mui/material/TableContainer';
...
-          <PaginationInWords
-            page={filters.page}
-            limit={filters.pageSize}
-            total_rows={total_rows}
-            offset={filters.offset}
-            isEscort={false}
-          />
-        </PaginationBox>
-      )}
+      <TableContainer sx={{ pb: 4, '& th': { py: 1 }, '& td': { py: 1.25 } }}>
```
Comment (WorkspaceChetan) [inline src/views/protectedDashboardViews/payoutsAndInvoicesTable/billingTable/BillingTable.tsx:13]: make style component

File: src/views/protectedDashboardViews/payoutsAndInvoicesTable/index.tsx [hunk 2]
```diff
@@ -56,7 +144,32 @@ const PayoutsAndInvoices = () => {
             </UINewTypography>
           </TypographyBox>
           <TableBox>
-            <BillingTable searchString={searchString} />
+            {isLoading ? (
+              <TableRow sx={{ border: 'none', display: 'flex', justifyContent: 'center' }}>
```
Comment (WorkspaceChetan) [inline src/views/protectedDashboardViews/payoutsAndInvoicesTable/index.tsx:148]: make style component

File: src/views/protectedDashboardViews/payoutsAndInvoicesTable/billingTable/BillingTable.tsx (hunk 1 above)
Comment (dhruv-divtech) [reply src/views/protectedDashboardViews/payoutsAndInvoicesTable/billingTable/BillingTable.tsx]: resolved

File: src/views/protectedDashboardViews/payoutsAndInvoicesTable/index.tsx (hunk 2 above)
Comment (dhruv-divtech) [reply src/views/protectedDashboardViews/payoutsAndInvoicesTable/index.tsx]: resolved

**Pull request merged**